src/styles/styles.js

```diff
@@ -2340,6 +2340,12 @@ const styles = {
         backgroundColor: colors.black,
         flex: 1,
     },
+    formHelp: {
```

Comment:
Please place it before the formError style.

Reply:
Also, there should be a newline before and after each style.

Reply:
Hmm if we're going for alphabetical styles it should go right below `formError` right?

Reply:
Aah looks like we're not really enforcing alphabetical styles much at all 😅 I retract my previous comment!